share/errmsg-utf8.txt

```diff
@@ -19930,6 +19930,12 @@ ER_REDO_ENCRYPTION_KEYRING
 ER_UNDO_NO_KEYRING
   eng "Undo log can't be encrypted if the keyring plugin is not loaded."
 
+ER_REDO_ENCRYPTION_CANT_FETCH_DEFAULT_KEY
+  eng "Couldn't fetch default percona_redo key. Please check if keyring is operational."
+
+ER_REDO_ENCRYPTION_CANT_FETCH_NOR_GENERATE_DEFAULT_KEY
+  eng "Couldn't fetch nor generate default percona_redo key. Please check if keyring is operational."
+
```

Comment:
may be simple ER_REDO_KEYRING_NO_KEY or CANT_FETCH? 

I dont see how the second message is useful to user. they are similar w.r.t to action taken by user.
Just one is sufficient IMO

Reply:
done